site/pages/what-we-do/header-slice/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styles from './style.css';
+import arrowImg from './arrow.png';
+
+const headerSlice = () => {
+  return (
+    <section className={styles.pageHeaderContainer}>
+      <div className={styles.left}>
+        <div className={styles.flexible}>
+          <h1 className={styles.pageHeader}>
+            <p>Do the</p>
+            <p>right thing</p>
```

Comment:
This is invalid HTML. `p` tags can only be used where flow content is expected. For an `h1` only phrasing content can be used https://html.spec.whatwg.org/dev/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements

Reply:
Thanks for the resource!